docs/core/runtime-config/index.md

```diff
@@ -101,11 +105,18 @@ Here is an example SDK-style project file with MSBuild properties for configurin
     <ThreadPoolMaxThreads>25</ThreadPoolMaxThreads>
   </PropertyGroup>
 
+  <ItemGroup>
+    <RuntimeHostConfigurationOption Include="System.Drawing.EnableUnixSupport" Value="true" />
+    <RuntimeHostConfigurationOption Include="System.Net.DisableIPv6" Value="true" />
+  </ItemGroup>
+
 </Project>
 ```
 
 MSBuild properties for configuring run-time behavior are noted in the individual articles for each area, for example, [garbage collection](garbage-collector.md). They are also listed in the [Runtime configuration](../project-sdk/msbuild-props.md#runtime-configuration-properties) section of the MSBuild properties reference for SDK-style projects.
 
+For runtime configuration settings that don't have a specific MSBuild property, the `RuntimeHostConfigurationOption` MSBuild item may be used instead.
```

Comment:
```suggestion
For runtime configuration settings that don't have a specific MSBuild property, you can use the `RuntimeHostConfigurationOption` MSBuild item instead.
```

Reply:
What do you think about moving this sentence to insert it at line 92?

Reply:
Good idea, done in dfb2cd